src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Machine banderomobil = new Truck();
+        Machine zil = new Bulldozer();
+        Machine gazon = new Exavator();
+
+        banderomobil.doWork();
+        zil.doWork();
+        gazon.doWork();
+
+        banderomobil.stopWork();
+        zil.stopWork();
+        gazon.stopWork();
```

Comment:
here better to create an array of Machines with Truck, Bulldozer and Excavator and call methods doWork and stopWork in a loop